gc/base/Bits.hpp

```diff
@@ -43,6 +44,44 @@ class MM_Bits {
 		return x << OMR_LOG_POINTER_SIZE;
```

Comment:
These two should be update to use logReferenceSize()

Reply:
They aren't the same - reference size is sizeof(fomrobject_t), where pointer size is sizeof(uintptr_t).

Reply:
ok, I see. 'slot' seems to be interchangeably used for both object slot, and native slot. 

perhaps we should be more specific in the name (nativeSlot or pointerSlot or ...?), but it's beyond the scope of this PR